### ISSUE TITLE
MIPS architecture support

### ISSUE THREAD

**Issue by vertexodessa** [preshing/turf#11]
Hi Jeff, 
this repository is awesome, thank you!

there is ARM and X86_64 support in turf/c/impl/, MIPS support is also something that would be nice to have.
I'm not sure if raising an issue would be the right way to start the discussion, please feel free to close the issue if you don't plan/want to add MIPS support in the future.

Thanks!

**Comment by preshing** [comment]
I agree that it would be great for Turf to include MIPS support! However I will need to rely on contributors for that.
